### PR TITLE
Display network in select in edit account modal

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

This PR displays crypto wallet details in the edit account modal. It also reverts to using a modal for the edit counterparty form and removes a page that was previously created.

Note: I'll create a separate story to improve the look of the edit counterparty modal when a scrollbar appears.

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/28436050?key=920d52cbfa2cef2a5665a5e767066be8

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: pkg/web/static/js/counterparties.js [hunk 1]
```diff
@@ -21,12 +21,36 @@ document.body.addEventListener('htmx:afterRequest', (e) => {
...
   select: '#countries',
   settings: {
     contentLocation: document.getElementById('country-content'),
   },
 });
-
 countriesArray.unshift({ 'placeholder': true, 'text': 'Select a country', 'value': '' });
-countrySelect.setData(countriesArray);
\ No newline at end of file
+countrySelect.setData(countriesArray);
```
Comment (daniellemaxwell) [inline pkg/web/static/js/counterparties.js:32]: This appears to be an organization change.

File: pkg/web/static/js/counterparties.js [hunk 2]
```diff
@@ -21,12 +21,36 @@ document.body.addEventListener('htmx:afterRequest', (e) => {
...
+document.body.addEventListener('htmx:afterSettle', (e) => {
+  const cpartyID = document.getElementById('cparty_id');
+  const cpartyPreviewEP = `/v1/counterparties/${cpartyID?.value}/edit`;
+
+  if (e.detail.requestConfig.path === cpartyPreviewEP && e.detail.requestConfig.verb === 'get') {
+    // Initialize SlimSelect for the country dropdown in the edit counterparty modal form.
+    const countrySelect = new SlimSelect({
+      select: '#country',
+      settings: {
+        contentLocation: document.getElementById('cparty_modal'),
```
Comment (daniellemaxwell) [inline pkg/web/static/js/counterparties.js:44]: It seems that adding an empty div for the content was too specific and setting the location to the modal solves the issue I was running into where the select options weren't appearing inside modals.

File: pkg/web/templates/partials/account/account_preview.html [hunk 3]
```diff
@@ -21,24 +22,29 @@ <h1 class="font-bold text-xl">Edit Customer Account</h1>
...
+      {{ end }}
     </div>
+    {{ end }}
     <div class="mb-4">
       <label for="tag" class="label-style">Notes</label>
       <textarea id="tag" name="tag" rows="4" class="block p-2.5 w-full text-sm text-gray-900 bg-gray-50 rounded-lg border border-gray-300 focus:ring-blue-500 focus:border-blue-500"></textarea>
     </div>
     <div class="flex justify-center">
-      <button hx-put="/v1/accounts/{{ .ID }}" class="w-44 btn bg-[#55ACD8] font-semibold text-lg text-white hover:bg-[#55ACD8]/90">Save</button>
+      <button hx-put="/v1/accounts/{{ .ID }}" class="w-44 btn bg-primary font-semibold text-lg text-white hover:bg-primary/90">Save</button>
```
Comment (daniellemaxwell) [inline pkg/web/templates/partials/account/account_preview.html:47]: This color was previously set in the tailwind config to replace HEX codes.

File: pkg/web/static/js/counterparties.js (hunk 2 above)
Comment (pdamodaran) [reply pkg/web/static/js/counterparties.js:44]: nice!

File: pkg/web/static/js/customerAccounts.js [hunk 4]
```diff
@@ -90,4 +90,37 @@ document.body.addEventListener('htmx:afterRequest', (e) => {
       walletDiv?.removeChild(walletDiv.lastChild)
     }
   }
-});
\ No newline at end of file
+});
+
```
Comment (pdamodaran) [inline pkg/web/static/js/customerAccounts.js:94]: Could you add a comment to describe what this function is doing?

File: pkg/web/static/js/customerAccounts.js [hunk 5]
```diff
@@ -90,4 +90,37 @@ document.body.addEventListener('htmx:afterRequest', (e) => {
...
+        const networkValue = selectedNetwork.value;
+        setNetworkData(network, networkValue);
+      };
+    });
+  };
+});
+
+// Set the network options and selected value in a SlimSelect dropdown for each network.
+function setNetworkData(el, value) {
+  el.slim.setData(networksArray);
```
Comment (pdamodaran) [inline pkg/web/static/js/customerAccounts.js:124]: `networksArray` is not passed in to this function

**Review by pdamodaran — APPROVED** [review]
I placed a couple of comments in the review but otherwise this is good to go!

File: pkg/web/static/js/customerAccounts.js (hunk 5 above)
Comment (daniellemaxwell) [reply pkg/web/static/js/customerAccounts.js:124]: `networksArray` is imported from `constants.js` and is passed into SlimSelect's `setData` method to modify the options for the element `(el)` passed into the `setNetworkData` function.

**Pull request merged**